vm_supervisor/models.py

```diff
@@ -120,6 +122,7 @@ async def create(
             await vm.start()
             await vm.configure()
             await vm.start_guest_api()
+            await self.run_code({"type": "lifespan.startup"})
```

Comment:
Why not launch this event from the start of the VM init1.py ?

Reply:
You mean from here? https://github.com/aleph-im/aleph-vm/blob/main/vm_supervisor/vm/firecracker_microvm.py#L133